3-platform/warehouse/.gitignore

```diff
@@ -0,0 +1,5 @@
+.git
```

Comment:
there are already present .gitignore file in root directory, e.g. 6core/.gitignore. 

Modify it, to apply your changes globally